pennylane_lightning/core/lightning_base.py

```diff
@@ -255,14 +255,16 @@ def _get_basis_state_index(self, state, wires):
         return int(qml.math.dot(state, basis_states))
 
     # pylint: disable=too-many-function-args, assignment-from-no-return
-    def _process_jacobian_tape(self, tape, starting_state, use_device_state):
+    def _process_jacobian_tape(
+        self, tape, starting_state, use_device_state, used_mpi: bool = False
```

Comment:
Nice catch!

Reply:
Agreed

Reply:
As a follow on, this is another location I think we want to avoid MPI-specifics. We should think on a design that better allows us to abstract this information into another way to be passed to where it is required.

Reply:
Done!